pkg/web/static/openapi.json

```diff
@@ -1520,6 +1817,249 @@
             "format": "base64"
           }
         }
+      },
+      "Rejection": {
+        "type": "object",
+        "properties": {
+          "code": {
+            "type": "string",
+            "enum": [
+              "REJECTED",
```

Comment:
The enum values have been updated to correspond with the what's listed in the UI and may be viewed [here](https://github.com/trisacrypto/envoy/blob/main/pkg/web/templates/components/transaction_rejection.html#L16-L30). A link to the all of the possible error codes has been added in a description. Let me know if you have any questions.